public_html/wp-content/mu-plugins/blocks/source/components/image/style.scss

```diff
@@ -1,4 +1,4 @@
-.wordcamp-image-container {
+.wordcamp-image__avatar-container,.wordcamp-image__featured-image-container {
```

Comment:
`SHOULD` - The second selector should be on a new line.

Reply:
Fixed along with lint stuff in https://github.com/WordPress/wordcamp.org/pull/123/commits/92ab54753af3ccb3e8248f03065b2e340521a261